addon/components/o-s-s/toggle-switch.stories.js

```diff
@@ -0,0 +1,42 @@
+import hbs from 'htmlbars-inline-precompile';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Components/OSS::ToggleSwitch',
+  component: 'toggle-switch',
+  argTypes: {
+    value: {
+      description: 'The value applied to the input/switch',
+      table: {
+        type: {
+          summary: 'boolean'
+        },
+        defaultValue: { summary: false }
+      },
+      control: { type: 'text' }
```

Comment:
```suggestion
      control: { type: 'boolean' }
```
The type of this value is boolean, let's define the same type for the `control`

Reply:
Indeed ! thanks, fixed.